website/docs/r/s3_bucket.html.markdown

```diff
@@ -318,15 +318,15 @@ resource "aws_s3_bucket" "bucket" {
   bucket = "mybucket"
 
   grant {
-    id         = "${data.aws_canonical_user_id.current_user.id}"
-    type       = "CanonicalUser"
-    permission = ["FULL_ACCESS"]
+    id          = "${data.aws_canonical_user_id.current_user.id}"
+    type        = "CanonicalUser"
+    permissions = ["FULL_ACCESS"]
```

Comment:
I also needed to rename `FULL_ACCESS` to `FULL_CONTROL` for this to work for me.